compiler/daml-extension/src/extension.ts

```diff
@@ -147,6 +151,37 @@ async function showReleaseNotes(version: string) {
     });
 }
 
+// Check if there is a new blog post which the user has not yet seen.
+// If so, display a notification with the link to the new post.
+// Update the user state so we don't notify about the same blog post again.
+// The user can opt out of these notifications entirely by changing the
+// 'daml.showNewBlogPosts' setting.
+async function showBlogIfNotSeen(config: WorkspaceConfiguration, context: ExtensionContext) {
+    if (!config.get('showNewBlogPosts')) { return; }
+    const feedUrl = 'https://blog.daml.com/daml-driven/rss.xml';
+    fetch(feedUrl).then(async (res: Response) => {
+        if (res.ok) {
+            const rssXml = await res.text();
+            parseString(rssXml, async function (err, rss) {
+                const latestBlog = rss.rss.channel[0].item[0];
+                const lastSeenBlog = context.globalState.get(recentBlogContextKey);
+                if (latestBlog &&
+                    (!lastSeenBlog || typeof lastSeenBlog === 'string' && lastSeenBlog !== latestBlog.title)) {
+                    window.showInformationMessage(
+                        `New blog post: ${latestBlog.title}`,
+                        'Go to blog'
+                    ).then(function(clicked) {
```

Comment:
Can we change this to something like:
```
There is a new post on the DAML Driven blog:
Another post why DAML is the best!
[Go to blog] [Ignore] [Snooze for one week] [Never show this again]
```

Reply:
Hmm I kind of like the simplicity of the current notification. Snooze is a good idea, but our blog posts are not being released that frequently at the moment - maybe if we were releasing things more than once a week then this would be a useful button. "Never show this again" might also be handy - in the PR as it stands you have to go to your settings to turn the notifications off - but maybe that added hurdle is a good thing to get users to stick with the feature for a little while at first. Anyway, I suggest we merge this change as is (probably after the style changes) and then consider adding more options.